.github/workflows/csharp.yml

```diff
@@ -8,64 +8,88 @@ on:
             - glide-core/**
             - submodules/**
             - .github/workflows/csharp.yml
+            - .github/workflows/install-shared-dependencies/action.yml
```

Comment:
I noticed there are some differences between the client CI workflows here. Do we need to align so that all clients have the same paths listed here and under pull_request?

Reply:
Yes
I added these references here, because they are used below